frontend/apps/app/app/api/webhook/github/utils/__tests__/checkSchemaChanges.test.ts

```diff
@@ -0,0 +1,123 @@
+import { getPullRequestFiles } from '@/libs/github/api.server'
+import { savePullRequestTask } from '@/src/trigger/jobs'
+import { prisma } from '@liam-hq/db'
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+import { checkSchemaChanges } from '../checkSchemaChanges'
+
+vi.mock('@/libs/github/api.server', () => ({
+  getPullRequestFiles: vi.fn(() =>
+    Promise.resolve([
+      {
+        filename: 'dummy.sql',
+        status: 'added',
+        additions: 1,
+        deletions: 0,
+        changes: 1,
+        fileType: 'text',
+      },
+    ]),
+  ),
+}))
+
+vi.mock('@/src/trigger/jobs', () => ({
+  savePullRequestTask: { trigger: vi.fn() },
+}))
+
+vi.mock('@liam-hq/db', () => ({
+  prisma: {
+    project: {
+      findUnique: vi.fn().mockImplementation(() => null),
+    },
+  },
+}))
```

Comment:
We want to run prisma in test environment... But it is good to deal with another issue.